kotlin-bundled-compiler/build.gradle.kts

```diff
@@ -196,7 +196,7 @@ val extractPackagesFromKTCompiler by tasks.registering(Jar::class) {
 }
 
 val downloadIntellijCoreAndExtractSelectedJars by tasks.registering {
-    dependsOn(deleteLibrariesFromLibFolder)
+    /*dependsOn(deleteLibrariesFromLibFolder)
```

Comment:
Did you intend to leave this commented out? Maybe add a comment...

Reply:
ah no i just forgot to remove it. Added that in a new commit.